# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1506801210 18000
#      Sat Sep 30 14:53:30 2017 -0500
# Node ID 0dc49e906fa5ab699ae7ca07c53aabf0fd4519b7
# Parent  6fdbdb66d7cb12a737efb81c48fea3bf34e3ca79
* liboctave/util/oct-rl-edit.h: Remove unused function declaration (patch #9459).

Pushed by Kai T. Ohlhus <k.ohlhus@gmail.com>.

diff --git a/liboctave/util/oct-rl-edit.h b/liboctave/util/oct-rl-edit.h
--- a/liboctave/util/oct-rl-edit.h
+++ b/liboctave/util/oct-rl-edit.h
@@ -63,18 +63,16 @@ extern void octave_rl_redisplay (void);
 extern int octave_rl_screen_height (void);
 
 extern int octave_rl_screen_width (void);
 
 extern void octave_rl_enable_paren_matching (int);
 
 extern int octave_rl_erase_empty_line (int);
 
-extern void octave_rl_init (void);
-
 extern void octave_rl_clear_screen (int skip_redisplay);
 
 extern void octave_rl_resize_terminal (void);
 
 extern void octave_rl_resize_terminal (void);
 
 extern void octave_rl_set_screen_size (int ht, int wd);
 
